src/routes/tools/extreme-precipitation/_store.js

```diff
@@ -0,0 +1,125 @@
+import { writable, derived } from "svelte/store";
+import { merge } from "d3-array";
```

Comment:
`merge` is not used so could be removed.

Reply:
removed in https://github.com/berkeley-gif/caladapt-website-2021/pull/211/commits/8ca43da57835471981e45e7209fb9c0211a061a3